jinja2/base.html

```diff
@@ -59,6 +59,7 @@
     <link rel="search" type="application/opensearchdescription+xml" href="{{ settings.SITE_URL }}/{{ request.LANGUAGE_CODE }}/search/xml" title="{{ _('MDN Web Docs') }}">
   {% endif %}
 
+  <meta name="lang-cookie" content="{{ settings. LANGUAGE_COOKIE_NAME }}">
```

Comment:
Whoops, I believe there is an unintended space between `settings.` and `LANGUAGE_COOKIE_NAME`?